src/private/MDMPresentationTransitionController.m

```diff
@@ -93,10 +93,14 @@ - (UIPresentationController *)presentationControllerForPresentedViewController:(
     return nil;
   }
   id<MDMTransitionWithPresentation> withPresentation = (id<MDMTransitionWithPresentation>)_transition;
-  _presentationController = [withPresentation presentationControllerForPresentedViewController:presented
-                                                                      presentingViewController:presenting
-                                                                          sourceViewController:source];
-  return _presentationController;
+  UIPresentationController *presentationController =
+      [withPresentation presentationControllerForPresentedViewController:presented
+                                                presentingViewController:presenting
+                                                    sourceViewController:source];
+  // _presentationController is weakly-held, so we have to do this local var dance to keep it
+  // from being nil'd on assignment.
```

Comment:
Unfortunately yes - I noticed that `_presentationController` was nil after assignment even though we were returning it from the function. I expect this is because the return statement isn't a sufficient indicator of a need for retainment in ARC's mind.